lib/gui/app/theme.ts

```diff
@@ -109,4 +111,11 @@ export const theme = {
 			}
 		`,
 	},
-};
+	layer: {
+		extend: () => `
+			> div:first-child {
+				background-color: transparent;
+			}
+		`,
+	},
```

Comment:
This isn't needed.

Reply:
It is for the modals layer background (which is not transparent by default)